app/registries/foreman/plugin/assets.rb

```diff
@@ -30,8 +30,13 @@ def webpack_manifest_path
         File.file?(manifest_path) ? manifest_path : nil
       end
 
+      def webpack_remote_entry
+        remote_entry_path = File.join(path, 'public', 'webpack', id.to_s, "#{id}_remoteEntry.js")
+        File.file?(remote_entry_path) ? remote_entry_path : nil
+      end
+
       def uses_webpack?
-        path && (File.file?(File.join(path, 'webpack', 'index.js')) || webpack_manifest_path.present?)
+        path && (File.file?(File.join(path, 'webpack', 'index.js')) || webpack_manifest_path.present? || webpack_remote_entry.present?)
```

Comment:
When is ` path && (File.file?(File.join(path, 'webpack', 'index.js'))` false? I only see it as true all the time

Reply:
when we use RPMs as we do not ship the `webpack` folder at all there, as we do not need to compile anything from it on the system where we deploy.